source/_integrations/knx.markdown

```diff
@@ -76,6 +76,21 @@ rate_limit:
   required: false
   default: 20
   type: integer
+individual_address:
+  description: The individual address.
```

Comment:
```suggestion
  description: The KNX individual address that shall be used. Note: If tunnelling server assigns an IA for a connection this will be used instead.
```